iOSClient/Main/Collection Common/NCCollectionViewCommon+CollectionViewDelegate.swift

```diff
@@ -82,6 +84,10 @@ extension NCCollectionViewCommon: UICollectionViewDelegate {
                                                                                                sceneIdentifier: self.controller?.sceneIdentifier) {
 
                 NCNetworking.shared.download(metadata: metadata, withNotificationProgressTask: true)
+
+                if metadata.url.isEmpty == false {
```

Comment:
```suggestion
                if !metadata.url.isEmpty {
```

Reply:
Fine with me. I usually prefer the more explicit expression because it cannot be overlooked as easily as the short form.